settings/settings.php

```diff
@@ -26,13 +26,15 @@ function register_block() {
  */
 function replace_core_ui_with_custom() : void {
 	/*
-	@todo Temporarily commented so that WebAuthn can be managed via wp-admin. Restore this when our custom WebAuthn UI is ready.
-	See https://github.com/WordPress/wporg-two-factor/issues/114, https://github.com/WordPress/wporg-two-factor/issues/87.
-	remove_action( 'show_user_profile', array( 'Two_Factor_Core', 'user_two_factor_options' ) );
-	remove_action( 'edit_user_profile', array( 'Two_Factor_Core', 'user_two_factor_options' ) );
-	remove_action( 'personal_options_update', array( 'Two_Factor_Core', 'user_two_factor_options_update' ) );
-	remove_action( 'edit_user_profile_update', array( 'Two_Factor_Core', 'user_two_factor_options_update' ) );
-	*/
+	 * @todo Temporarily commented so that WebAuthn can be managed via wp-admin. Restore this when our custom WebAuthn UI is ready.
```

Comment:
I think the comment needs to be updated to reflect the new intent:

```diff
- * @todo Temporarily commented so that WebAuthn can be managed via wp-admin. Restore this when our custom WebAuthn UI is ready.
+ * @todo Temporarily scoped to the front end, so that WebAuthn can be managed via wp-admin. Remove the conditional when our custom WebAuthn UI is ready, so that the front-end is the only place to manage it.
```